user/fcgiwrap/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "fcgiwrap"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+# without an empty prefix, everything gets installed to /usr/usr/
+configure_args = ["--prefix="]
+make_dir = "."
+hostmakedepends = ["automake", "pkgconf"]
+makedepends = ["fcgi-devel"]
+pkgdesc = "Simple server for running CGI applications over FastCGI"
+maintainer = "mia <mia@mia.jetzt>"
+license = "MIT"
+url = "https://github.com/gnosek/fcgiwrap"
+source = [
+    f"{url}/archive/{pkgver}.tar.gz",
+    # final tag doesn't include license
+    f"!{url}/blob/99c942c90063c73734e56bacaa65f947772d9186/COPYING",
```

Comment:
this should be `https://raw.githubusercontent.com/gnosek/fcgiwrap/b6696a1495dc818d0b22d9c2e9be7c618e25b3af/COPYING` or it will fetch an html and that will have a different checksum every time